crafters/image/ImageTorchTransformation/__init__.py

```diff
@@ -0,0 +1,168 @@
+from typing import Dict, List, Union
+
+import numpy as np
+import torch
```

Comment:
can you please hide this `import` under:

```
if False:
    import torch
```

just to offer access for type checking, and do the imports only where they are needed? This way, we do not pollute the `jina check` command line

Reply:
You are right forgot about that, thanks